client/lib/mixins/i18n/index.js

```diff
@@ -259,6 +259,10 @@ function getTranslationFromJed( options ) {
 	var jedMethod = 'gettext',
 		jedArgs;
 
+	if ( ! i18nState.jed ) {
```

Comment:
>I have no experience with this lib so I also have no idea if that change is safe :)

My confidence in it is driven by the knowledge that if it had reached this point without `i18nState.jed` being defined, it was on track to explode anyways :laughing: (given the property function call a few lines down)

>Would it make sense to refactor test to use exported plain component and simply pass `identity` as `translate` prop?

Yeah, I think that'd be a good approach for testing components. This is a bit of an odd case where the module being tested itself renders the component. Perhaps because it's such an odd case I should be responsible for either initializing i18n or mocking the underlying component. I'll take another look.